app/src/main/java/com/globant/example/mentorapp/presentation/di/Module/ListUsersModule.java

```diff
@@ -0,0 +1,128 @@
+package com.globant.example.mentorapp.presentation.di.Module;
+
+import android.arch.lifecycle.MutableLiveData;
+import android.content.Context;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+
+import com.globant.example.mentorapp.MentorApplication;
+import com.globant.example.mentorapp.data.entity.UserEntity;
+import com.globant.example.mentorapp.data.remote.APIClient;
+import com.globant.example.mentorapp.data.util.ApiUtils;
+import com.globant.example.mentorapp.domain.interactor.ListUsersInteractorImpl;
+import com.globant.example.mentorapp.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.presentation.presenter.ListUsersPresenterImpl;
+import com.globant.example.mentorapp.presentation.view.fragment.ListUsersFragment;
+import com.squareup.otto.Bus;
+import com.squareup.otto.ThreadEnforcer;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Class to control UserList injections
+ * Created by oscar.amezquita on 8/06/2017.
+ */
+@Module
+public class ListUsersModule {
```

Comment:
This name seems to be very specific even though it provides different objects that are not related to a list of users. Use a generic name seems this is the main module of the app: ApplicationModule. (This can also apply for the component: ApplicationComponent)